src/o365/spfx/commands/project/project-externalize/model/IVisitationResult.ts

```diff
@@ -0,0 +1,7 @@
+import { FileEditSuggestion } from "./FileEditSuggestion";
+import { ExternalizeEntry } from "./ExternalizeEntry";
+
+export interface IVisitationResult {
+  entries: ExternalizeEntry[];
+  suggestions: FileEditSuggestion[]
```

Comment:
Since everything we show is a suggestion/recommendation, how about we rename the type to `FileEdit`?

Reply:
renamed